src/ui/pages/equipment/location/Location.js

```diff
@@ -230,7 +250,8 @@ export default class Location extends Entity {
                 },
                 column: 2,
                 order: 11,
-                ignore: !isCernMode
+                ignore: !isCernMode,
+                initialVisibility: false
```

Comment:
Is this `initialVisibility` value intended?

Reply:
Location-> Equipment Graph does not have a corresponding tab. Perhaps a better option is to remove it altogether.